docs/tutorials/notebooks/EntropicMaps_Barriere_Peltier.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    entropic_pot = potentials.EntropicPotentials(
</pre><p>this is easier by just instantiating <code>ot_sink.to_dual_potentials</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>